src/FakeItEasy/ExceptionMessages.resx

```diff
@@ -192,17 +192,6 @@ The following constructors failed:
   <data name="FakeCreationExceptionDefaultMessage" xml:space="preserve">
     <value>Unable to create fake object.</value>
   </data>
-  <data name="FailedToGenerateProxyPattern" xml:space="preserve">
-    <value>FakeItEasy failed to create fake object of type "{0}".
-
-1. The type is not registered in the current IFakeObjectContainer.
-2. The current IProxyGenerator failed to generate a proxy for the following reason:
-
-{1}</value>
-  </data>
-  <data name="UnableToCreateDummyPattern" xml:space="preserve">
-    <value>FakeItEasy was unable to create dummy of type "{0}", register it in the current IFakeObjectContainer to enable this.</value>

```

Comment:
I am not sure from your response whether my previous message was clear. Sorry if there was confusion.
The removals are good, and as you point out, completely related to this PR. I'd be happy to have them in this PR, and there's no need to make a separate commit.